utils.py

```diff
@@ -405,13 +412,23 @@ def get_time_in_millisecs(datetime_obj):
 
 
 def get_current_time_in_millisecs():
-    """Returns time in milliseconds since the Epoch."""
+    """Returns time in milliseconds since the Epoch.
+    
+    Return:
```

Comment:
Returns* (Follow the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python).)

Reply:
done